### DISCUSSION TITLE
Is there a way to not minify files generated with svelte-package?

### DISCUSSION THREAD

**Discussion by raythurnvoid** [sveltejs/kit#9116] [Q&A] [answered]
Is there a way to not minify files generated with svelte-package?

**Comment by dummdidumm** [comment, marked as answer]
What do you mean by this? They are not minified.

**Reply by raythurnvoid** [reply]
You are right, i got confused because of the output files being in commonjs syntax: https://github.com/sveltejs/kit/discussions/9117#discussioncomment-5063037
